packages/glimmer-syntax/lib/traversal/walker.ts

```diff
@@ -1,28 +1,3 @@
-function Walker(order=undefined) {
-  this.order = order;
-  this.stack = [];
-}
-
-export default Walker;
-
-Walker.prototype.visit = function(node, callback) {
-  if (!node) {
-    return;
-  }
-
-  this.stack.push(node);
-
-  if (this.order === 'post') {
-    this.children(node, callback);
-    callback(node, this);
-  } else {
-    callback(node, this);
-    this.children(node, callback);
-  }
-
-  this.stack.pop();
-};
-
 let visitors = {
   Program: function(walker, node, callback) {
```

Comment:
You can use shorthand syntax here and change to `Program(walker, node, callback) {`